parser/src/command.rs

```diff
@@ -140,7 +147,7 @@ impl<'a> Input<'a> {
         self.parsed += if c.is_ok() {
             tok.position()
         } else {
-            self.bot.len() + 1
+            name_length
```

Comment:
@Mark-Simulacrum 
When the command is successfully parsed this correctly advances the input, but when it isn't, the input is now advanced by the number of names this bot has. See line 170:
https://github.com/rust-lang/triagebot/blob/f346181e014954f66c5c495a753522e0adc343fa/parser/src/command.rs#L166-L170

Reply:
Happy to receive a PR, I wasn't sure on the math myself tbh.

Reply:
Is there any reason to skip the whole bot name, not just the initial `@` ? Doing so would be easier and should still prevent the bot from matching again the same command.

Reply:
hm I don't think so, no